.github/ionic-issue-bot.yml

```diff
@@ -131,6 +134,8 @@ noReproduction:
   maxIssuesPerRun: 100
   label: "ionitron: needs reproduction"
   responseLabel: triage
+  exemptLabels:
+    - "needs: investigation"
```

Comment:
```suggestion
```

This isn't needed because the issue is only closed if it has `ionitron: needs reproduction` and we don't add that to investigative tickets